psalm-baseline.xml

```diff
@@ -836,6 +840,9 @@
       <code><![CDATA[Filter\Decrypt]]></code>
       <code><![CDATA[parent::filter($content)]]></code>
     </DeprecatedClass>
+    <DeprecatedInterface>
```

Comment:
unsure why theses entries are added. I only added the deprecations and run `psalm --set-baseline=psalm-baseline.xml` with the psalm version of the lock file

Reply:
`--no-cache` did the trick